src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx

```diff
@@ -148,12 +148,12 @@ const CreateProjectFormView: React.FC<CreateProjectFormViewProps> = ({ allowSubm
           </Grid>
         </Grid>
         <Box display="flex" gap={2} sx={{ mt: 2 }}>
```

Comment:
remove gap = {2} here it's adding another 2 px gap between the buttons, so it's 4 px apart total